### DISCUSSION TITLE
Toggle modal with button instead of link

### DISCUSSION THREAD

**Discussion by taiwanleaftea** [orchidsoftware/platform#2463] [Q&A] [answered]
By default this code generates a link

`ModalToggle::make('Launch demo modal')->modal('exampleModal')->method('action')->icon('full-screen')`

Is it possible to generate modal toggler as a button?

**Comment by c-gross** [comment, marked as answer]
ModalToggle is a button (`class ModalToggle extends Button`)

Guess you want it to look more like a button?
Try

```
ModalToggle::make('Launch demo modal')
->modal('exampleModal')
->method('action')
->class('btn btn-success')
->icon('full-screen')
```

Adjust the css button class as you need.

**Comment by jfuentesa** [comment]
Hi:

Is it possible to generate modal opener as a text link? :)

Thank you.
